marshmallow_jsonapi/schema.py

```diff
@@ -76,7 +76,22 @@ class Meta:
         pass
 
     def __init__(self, *args, **kwargs):
+        self.include_data = kwargs.pop('include_data', ())
         super(Schema, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
+            if field_name in self.include_data:
+                if not isinstance(field, BaseRelationship):
+                    raise ValueError('Can only include relationships. "%s" is a "%s"'
+                                     % (field_name, field.__class__.__name__))
+                if not hasattr(field, 'schema') or not field.schema:
+                    raise ValueError('A schema is required to serialize "%s"' % field_name)
+                field.include_data = True
+            else:
+                field.include_data = False
```

Comment:
This will raise an error for field's that define `__slots__` (which is proposed here: https://github.com/marshmallow-code/marshmallow/pull/422). 

Can we safely remove this `else` block?